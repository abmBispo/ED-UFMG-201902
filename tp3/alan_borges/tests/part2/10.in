64
o mesmo do s do c e do z o mesmo do k e do g etc sao trapalhices caligraficas veja os algarismos nao ha dous que facam o mesmo oficio e e e e admire a beleza com que um e um formam esta cousa que se exprime por agora dobre e tera multiplique por igual numero da e assim por diante mas
c e
c do
c o
c por
c que
c mesmo
c um
c a
c admire
c agora
c algarismos
c assim
c beleza
c c
c caligraficas
c com
c cousa
c da
c diante
c dobre
c dous
c esta
c etc
c exprime
c facam
c formam
c g
c ha
c igual
c k
c mas
c multiplique
c nao
c numero
c oficio
c os
c s
c sao
c se
c tera
c trapalhices
c veja
c z
