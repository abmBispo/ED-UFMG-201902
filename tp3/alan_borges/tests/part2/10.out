110
1110
0000
0001
0010
11111
00110
001110
001111
010000
010001
010010
010011
010100
010101
010110
010111
011000
011001
011010
011011
011100
011101
011110
011111
100000
100001
100010
100011
100100
100101
100110
100111
101000
101001
101010
101011
101100
101101
101110
101111
111100
111101
